src/handlers/notification/index.js

```diff
@@ -334,11 +334,11 @@ const processMessage = async (msg, span) => {
         jwsSigner = getJWSSigner(ENUM.Http.Headers.FSPIOP.SWITCH.value)
         rv = await Callback.sendRequest(callbackURLFrom, callbackHeaders, ENUM.Http.Headers.FSPIOP.SWITCH.value, from, ENUM.Http.RestMethods.PUT, payloadForCallback, ENUM.Http.ResponseTypes.JSON, span, jwsSigner)
       } catch (err) {
-        histTimerEndSendRequest2({ success: false, to, dest: from, action, status: response.status })
+        histTimerEndSendRequest2({ success: false, dest: from, action, status: response.status })
```

Comment:
Please remove the `to` label from line 330. I believe that it should not be there to start off with --> https://github.com/mojaloop/ml-api-adapter/pull/437/files#diff-f4a4d7f564412cafc9274bd4f7b20930R330

Reply:
hi @mdebarros removed it.